src/base/__tests__/Context.ts

```diff
@@ -1174,13 +1182,13 @@ describe('Context class', () => {
         }
       );
 
-      let result = await context.issuedClaims({
+      let result = (await context.issuedClaims({
         targets: [targetDid],
         trustedClaimIssuers: [targetDid],
         claimTypes: [ClaimType.Accredited],
         includeExpired: true,
         size: 1,
-      });
+      })) as ResultSet<ClaimData>;
```

Comment:
These assertions shouldn't be necessary once the function is returning a single type